cyclops-ui/src/components/pages/EditModule.tsx

```diff
@@ -138,7 +132,7 @@ const EditModule = () => {
 
   let { moduleName } = useParams();
 
-  const mapsToArray = (fields: any[], values: any): any => {
+  const mapsToArray = useCallback((fields: any[], values: any): any => {
```

Comment:
`useCallback` will return a memoized version of the callback that only changes if one of the `inputs` has changed.